packages/backend/src/build-disk-image.ts

```diff
@@ -58,6 +58,13 @@ export async function buildDiskImage(build: BootcBuildInfo, history: History): P
     throw new Error('The podman machine is not set as rootful.');
   }
 
+  const isPodmanV5 = await machineUtils.isPodmanV5Machine();
+  if (!isPodmanV5) {
+    const errorMessage = 'Podman v5.0 or higher is required to build disk images.';
+    await extensionApi.window.showErrorMessage(errorMessage);
+    throw new Error('The podman machine version is below v5.0.');
```

Comment:
```suggestion
    throw new Error('Podman Machine is below v5.0');
```

Small syntax suggestion

Reply:
Fixed in new commit.